voyager_archive/views.py

```diff
@@ -25,7 +25,12 @@ def search(request: HttpRequest) -> None:
             elif search_type == 'MFHD_ID':
                 marc_record = get_mfhd_record(search_term)
 
-            return render(request, 'voyager_archive/search.html', {'form': form})
+            context = {
+                'form' : form,
```

Comment:
`flake8` reminder: no whitespace before colons in dictionaries, on these 2 lines